pilaf/.yarnrc

```diff
@@ -0,0 +1,9 @@
+nodeLinker: node-modules
```

Comment:
This should be in a `.yarnrc.yml` file but since the settings are inherited recursively from the parent folder you don't actually need this file at all since the one at the root specifies the same settings

Reply:
too late lol